src/library_getvalue.js

```diff
@@ -4,80 +4,61 @@
  * SPDX-License-Identifier: MIT
  */
 
+function setValueImpl(ptr, value, type = 'i8') {
```

Comment:
Maybe add a comment, `Note that we should not use the alias mechanism below, `setValue_safe = 'setValue'` because we actually do need two copies of this function code (so that the safe one is property handled by the SAFE_HEAP logic, but the other is treated like any other function).